### ISSUE TITLE
missing #include<cstring>

### ISSUE THREAD

**Issue by hv-work** [OpenSpace/Ghoul#45]
`ghoul/tests/test_crc32.cpp` and `include/ghoul/misc/memorypool.h` are missing `#include<cstring>` required for `strlen` and `memset`:
```
[ 94%] Building CXX object ext/ghoul/tests/CMakeFiles/GhoulTest.dir/test_crc32.cpp.o
/root/OpenSpace/ext/ghoul/tests/test_crc32.cpp: In function 'void ____C_A_T_C_H____T_E_S_T____0()':
/root/OpenSpace/ext/ghoul/tests/test_crc32.cpp:218:39: error: 'strlen' was not declared in this scope
             static_cast<unsigned int>(strlen(d.string))
                                       ^~~~~~
/root/OpenSpace/ext/ghoul/tests/test_crc32.cpp:218:39: note: 'strlen' is defined in header '<cstring>'; did you forget to '#include <cstring>'?
/root/OpenSpace/ext/ghoul/tests/test_crc32.cpp:29:1:
+#include <cstring>
 #include <random>
```
```
In file included from /root/OpenSpace/ext/ghoul/include/ghoul/misc/memorypool.h:191,
                 from /root/OpenSpace/ext/ghoul/tests/test_memorypool.cpp:28:
/root/OpenSpace/ext/ghoul/include/ghoul/misc/memorypool.inl: In member function 'void* ghoul::MemoryPool<BucketSize>::alloc(int)':
/root/OpenSpace/ext/ghoul/include/ghoul/misc/memorypool.inl:90:18: error: 'memset' is not a member of 'std'
             std::memset(reinterpret_cast<std::byte*>(ptr) + ii, 0xAB, 1);
```

g++ (Ubuntu 8.3.0-6ubuntu1~18.04.1) 8.3.0

**Comment by alexanderbock** [comment]
Fixed in 423d026fa489b0f8b53284be916f3d82bc33d3fd and OpenSpace/OpenSpace@fde307ead976fede7fe851eebd420948c15b524b

**Comment by alexanderbock** [comment]
Thanks for the issue reporting!
